src/interfaces/store-interfaces/state-interfaces/continuation-in-state-interface.ts

```diff
@@ -4,4 +4,5 @@ export interface ContinuationInStateIF {
   continuationAuthorizationPageValid: boolean
   authorizationProof: AuthorizationProofIF
   existingBusinessInfo: ExistingBusinessInfoIF
+  consumerDocumentId: string
```

Comment:
Which document is this for? In a continuation in there could be multiple authorization proof files and one optional  affidavit file. See AuthorizationProofIF and ExistingBusinessInfoIF.

Reply:
I gather this is for all authorization proof files.